integration-tests/modules/__tests__/inventory/index.spec.ts

```diff
@@ -608,145 +609,148 @@ medusaIntegrationTestRunner({
         })
       })
 
-      it.skip("should remove associated levels and reservations when deleting an inventory item", async () => {
-        const inventoryService = appContainer.resolve("inventoryService")
-
-        const invItem2 = await inventoryService.createInventoryItem({
-          sku: "1234567",
+      describe("delete inventory item", () => {
+        let invItem
+        beforeEach(async () => {
+          invItem = await service.create({
+            sku: "MY_SKU",
+            origin_country: "UK",
+            hs_code: "hs001",
+            mid_code: "mids",
+            material: "material",
+            weight: 300,
+            length: 100,
+            height: 200,
+            width: 150,
+          })
         })
 
-        const stockRes = await api.post(
-          `/admin/stock-locations`,
-          {
-            name: "Fake Warehouse 1",
-          },
-          adminHeaders
-        )
-
-        locationId = stockRes.data.stock_location.id
-
-        const level = await inventoryService.createInventoryLevel({
-          inventory_item_id: invItem2.id,
-          location_id: locationId,
-          stocked_quantity: 10,
-        })
+        it("should remove associated levels and reservations when deleting an inventory item", async () => {
+          const inventoryService = appContainer.resolve("inventoryService")
```

Comment:
suggestion: we should try to use our enums when resolving and for other things too when possible to not rely on string and be future proof to changes :) take or leave